### ISSUE TITLE
app crashed on apple tv (ios version is 12)

### ISSUE THREAD

**Issue by Sarvarr** [react-native-community/react-native-tvos#97]
Thread 1: Exception: "-[UITraitCollection hasDifferentColorAppearanceComparedToTraitCollection:]: unrecognized selector sent to instance 0x283d44900"

**Comment by dlowder-salesforce** [comment]
@Sarvarr please provide an example project that reproduces this issue, and specify which tvOS version you saw this on (12.0, 12.1, 12.2, 12.3, 12.4).

**Comment by Sarvarr** [comment]
@dlowder-salesforce
 tvOS:12.1, 
"react-native": "npm:react-native-tvos@0.63.1-0"

**Comment by dlowder-salesforce** [comment]
I just tested the latest 0.63.1 release branch and this crash is not happening for me on 12.1.

I believe it is fixed by commit https://github.com/react-native-community/react-native-tvos/commit/8fc3c2f635b1458fc6ee7531904e29fcb97b7eb5

It will be in the 0.63.1-1 release that is coming this week.

**Comment by dlowder-salesforce** [comment]
Confirmed: I backed out the above commit and tested on a 12.1 simulator, and do indeed see your crash:

```
2020-09-09 09:57:10.747694-0700 HelloWorld-tvOS[92746:3791125] -[UITraitCollection hasDifferentColorAppearanceComparedToTraitCollection:]: unrecognized selector sent to instance 0x600003e25880
2020-09-09 09:57:10.765765-0700 HelloWorld-tvOS[92746:3791125] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[UITraitCollection hasDifferentColorAppearanceComparedToTraitCollection:]: unrecognized selector sent to instance 0x600003e25880'
*** First throw call stack:
(
	0   CoreFoundation                      0x000000010a7b671b __exceptionPreprocess + 331
	1   libobjc.A.dylib                     0x00000001095d2735 objc_exception_throw + 48
	2   CoreFoundation                      0x000000010a7d54a4 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
	3   CoreFoundation                      0x000000010a7bb436 ___forwarding___ + 1446
	4   CoreFoundation                      0x000000010a7bd308 _CF_forwarding_prep_0 + 120
	5   HelloWorld-tvOS                     0x0000000106dccb15 -[RCTView traitCollectionDidChange:] + 149
	6   UIKitCore                           0x000000011255f126 -[UIView _wrappedProcessTraitCollectionDidChange:forceNotification:] + 137
	7   UIKitCore                           0x000000011255f233 -[UIView _wrappedProcessTraitCollectionDidChange:forceNotification:] + 406
```

I'll close this once the 0.63.1-1 release is out.

**Comment by dlowder-salesforce** [comment]
Fixed in 0.63.1-1
